ILSpy/Commands/GeneratePdbContextMenuEntry.cs

```diff
@@ -105,6 +116,109 @@ internal static void GeneratePdbForAssembly(LoadedAssembly assembly, LanguageSer
 				return output;
 			}, ct)).Then(dockWorkspace.ShowText).HandleExceptions();
 		}
+
+		internal static void GeneratePdbForAssemblies(System.Collections.Generic.IEnumerable<LoadedAssembly> assemblies, LanguageService languageService, DockWorkspace dockWorkspace)
+		{
+			var assemblyArray = assemblies?.Where(a => a != null).ToArray();
```

Comment:
```suggestion
			var assemblyArray = assemblies?.Where(a => a != null).ToArray() ?? [];
```

then you can omit the null checks later.

also I would assume that nobody passes null in here... at some point this needs nullable annotations